sagemaker/launch/launch_gconstruct.py

```diff
@@ -105,9 +104,7 @@ def get_gconstruct_parser():
 
     gconstruct_arguments = parser.add_argument_group("GraphStorm Gconstruct arguments")
 
-    gconstruct_arguments.add_argument("--input-graph-s3", type=str,
-        required=True, help="S3 location of the input graph data")
-    gconstruct_arguments.add_argument("--output-graph-s3", type=str,
+    gconstruct_arguments.add_argument("--output-graph-s3", "--output-data-s3", type=str,
```

Comment:
Is this a breaking change? If yes, please flag it and update the PR description.

Reply:
It's not breaking, we're just adding an alias to the call to make the gconstruct arguments consistent with the other launch scripts. Other scripts use `graph-data-s3` and `output-data-s3`, only gconstruct uses `input-graph-s3` and `output-graph-s3`. So adding an alias here and in the common parser for the input so both are supported.